CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 1.14.0 (May 29, 2024)
+
+ * [#4101](https://github.com/redhat-developer/vscode-openshift-tools/issues/4101) Project not listed if it has a display name annotation.
+ * [#4092](https://github.com/redhat-developer/vscode-openshift-tools/issues/4092) Trying to create a serverless function using the Spring Boot template causes the webview to fail.
+ * [#4086](https://github.com/redhat-developer/vscode-openshift-tools/issues/4086) Create an web view for supporting to create routes from the extension .
+ * [#4083](https://github.com/redhat-developer/vscode-openshift-tools/issues/4083) Helm chart browser broken.
+ * [#4064](https://github.com/redhat-developer/vscode-openshift-tools/pull/4064) Add a proposal message to change stdout max buffer length preference in case of 'ERR_CHILD_PROCESS_STDIO_MAXBUFFER' error appearance.
+ * [#4059](https://github.com/redhat-developer/vscode-openshift-tools/pull/4059) Get rid of using k8s extension provided 'kubectl' binary.
+ * [#3999](https://github.com/redhat-developer/vscode-openshift-tools/issues/3999) Make possible to work with projects/namespaces on clusters where the listing of projects is restricted.
+ * [#3990](https://github.com/redhat-developer/vscode-openshift-tools/issues/3990) Migrate Kubernetes extension commands.
+ * [#3987](https://github.com/redhat-developer/vscode-openshift-tools/issues/3987) App. Explorer "hangs" on extension loading if logged out from a cluster.
+ * [#3971](https://github.com/redhat-developer/vscode-openshift-tools/issues/3971) Re-logging in does not ask for username and password.
+ * [#3925](https://github.com/redhat-developer/vscode-openshift-tools/issues/3925) Fixed 'stdout maxBuffer length exceeded' error.
```

Comment:
I would probably group this with the entry that talks about the proposal message for max buffer exceeded.